elements/campaign-product-shot/campaign-product-shot.js

```diff
@@ -0,0 +1,50 @@
+import {
+  registerElement,
+  BulbsHTMLElement,
+} from 'bulbs-elements/register';
+import { filterBadResponse, getResponseJSON } from 'bulbs-elements/util';
+import invariant from 'invariant';
+
+class CampaignProductShot extends BulbsHTMLElement {
+
+  handleRequestSuccess (data) {
+    this.innerHTML =
+      `<div class='campaign-product-shot'>
+        <img src='${data.product_shot_url}'>
```

Comment:
We should check to make sure the `product_shot_url` is not empty prior to setting it on the img src.  Possibly log an error if so